bodhi/server/models.py

```diff
@@ -1671,6 +1676,17 @@ def get_changelog(self, timelimit=0):
         if not isinstance(when, list):
             when = [when]
 
+        if lastupdate:
+            lastpkg = self.get_latest()
+            if lastpkg is not None:
+                oldh = get_rpm_header(lastpkg)
+                if oldh['changelogtext']:
```

Comment:
Should we use get here ? or are we sure that the `changelogtext` key will always be there ?
```suggestion
                if oldh.get('changelogtext'):
```

Reply:
mmm I need to check if `changelogtext` is always present or not. Here I simply reused the old code that was in `bodhi.utils`.

Reply:
I've just checked, `changelogtext` will always be there